pkg/grpc/atttestation-api.proto

```diff
@@ -5,21 +5,15 @@ option go_package = "github.com/DIMO-Network/attestation-api/pkg/grpc";
 package grpc;
 
 service AttestationService {
-  rpc BatchCreateVINVC(BatchCreateVINVCRequest) returns (BatchCreateVINVCResponse);
+  rpc CreateVinVc(CreateVinVcRequest) returns (CreateVinVcResponse);
 }
 
 
-message BatchCreateVINVCRequest {
-  repeated uint32 tokenIds = 1;
+message CreateVinVcRequest {
+  uint32 token_id = 1;
   bool force = 2;
 }
 
-message BatchCreateVINVCResponse {
-  repeated VINVCResult results = 1;
+message CreateVinVcResponse {
+  string raw_vc = 1;
```

Comment:
Just returning the raw string. I don't really want to build another RPC call into Telemetry. We'll get rid of all this soon.